cypress.json

```diff
@@ -1,3 +1,3 @@
-{
-    "baseUrl": "https://giftaid-staging.comicrelief.com"
+{   //this is the temp envt, should change this to localhost later
+    "baseUrl": "https://5ba4dbef1f12b704899145bd--giftaid-react.netlify.com/update"
```

Comment:
Hmmm, I think `http://localhost:3000` will work everywhere, regardless of domain name